emission/core/wrapper/compositetrip.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import emission.core.wrapper.confirmedtrip as ecwc
+import emission.core.wrapper.wrapperbase as ecwb
+import emission.core.wrapper.modeprediction as ecwm
+
+class Compositetrip(ecwc.Confirmedtrip):
+    props = ecwc.Confirmedtrip.props
+    props.update({#      # confirmedplace stuff
+                  "confirmed_place": ecwb.WrapperBase.Access.WORM, # object contains all properties for the destination confirmed_place object
+                  "locations": ecwb.WrapperBase.Access.WORM, # object containing cleaned location entries (max 100)
+                  #      # sections stuff
+                  "cleaned_section": ecwb.WrapperBase.Access.WORM,
+                  "inferred_section": ecwb.WrapperBase.Access.WORM,
+                  "inferred_mode": ecwb.WrapperBase.Access.WORM, # inferred by mode inference algo
```

Comment:
typically, the mode is inferred at the section level, not the trip level, so that we can support multi-modal trips. So this should really be an array or a dict instead of a single field.

Reply:
These fields don't do anything yet but @sebastianbarry and I were speculating about how it might look.
I am not familiar enough with it yet; I'd like to revisit it in context when we actually add this to the label screen